test/libsolidity/syntaxTests/nameAndTypeResolution/488_function_types_selector_5.sol

```diff
@@ -2,9 +2,8 @@ contract C {
     function h() pure external {
     }
     function f() view external returns (bytes4) {
-        var g = this.h;
+        function() pure external g = this.h;
```

Comment:
Perhaps it would be better to use `this.h.selector`, because the type might change now during the assignment.

Reply:
I just noticed right now, that this test (`...selector_5`) is now equal to `...selector_6.sol` test, hence, could be straight eliminated.

What do you mean the type "might change now during the assignment"?

Reply:
The type of `this.h` and `g` might be different and `this.h` might not have a member called `selector` but `g` might.